src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -72,46 +78,19 @@ class ElectronicDeliveryForm extends React.Component {
           <legend>Contact Information</legend>
           <h3>Contact Information</h3>
 
-          <div className={`nypl-text-field ${errorClass.name}`}>
-            <label htmlFor="patron-name" id="patron-name-label">First and last name
-              <span className="nypl-required-field">&nbsp;(Required)</span>
-            </label>
-            <input
-              id="patron-name"
-              type="text"
-              required
-              aria-labelledby="patron-name-label patron-name-status"
-              aria-required="true"
-              name="name"
-              value={this.state.form.name}
-              onChange={(e) => this.handleUpdate(e, 'name')}
-            />
-            {
-              errorClass.name &&
-                (<span
-                  className="nypl-field-status"
-                  id="patron-name-status"
-                  aria-live="assertive"
-                  aria-atomic="true"
-                >
-                  Please enter your first and last name
-                </span>)
-            }
-          </div>
-
           <div className={`nypl-text-field ${errorClass.email}`}>
             <label htmlFor="email" id="email-label">Email address
```

Comment:
You forgot to update `htmlFor` to `email-address` since you updated the id for the input.